internal/service/sso/resource_identity_provider_test.go

```diff
@@ -365,12 +370,90 @@ func TestAccIdentityProvider_LinkedIn(t *testing.T) {
 				),
 			},
 			{
-				Config: testAccIdentityProviderConfig_LinkedIn2(resourceName, name),
+				Config: testAccIdentityProviderConfig_LinkedIn2(resourceName, name, "linkedin"),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(resourceFullName, "facebook.%", "0"),
 					resource.TestCheckResourceAttr(resourceFullName, "google.%", "0"),
 					resource.TestCheckResourceAttr(resourceFullName, "linkedin.client_id", "dummyclientid2"),
 					resource.TestCheckResourceAttr(resourceFullName, "linkedin.client_secret", "dummyclientsecret2"),
+					resource.TestCheckResourceAttr(resourceFullName, "linkedin_oidc.%", "0"),
+					// resource.TestMatchResourceAttr(resourceFullName, "linkedin.callback_url", regexp.MustCompile(`^https:\/\/auth\.pingone\.(?:eu|com|asia|ca)\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\/rp\/callback\/linkedin$`)),
```

Comment:
Is this supposed to be commented?

Reply:
I'm unsure. The comment line was there before I started this work - I'll check.